### ISSUE TITLE
App no longer works in Django 5.1.0

### ISSUE THREAD

**Issue by MaoMaoCake** [theriverman/django-minio-backend#54]
Due to new checks for `STATIC_ROOT` variable in 5.1.0 (not sure of exact version this was introduced but django 5.0.0 works) the lib is no longer able to copy to minio

Steps to reproduce:
clone the repository: `git clone https://github.com/theriverman/django-minio-backend.git && cd django-minio-backend`
instll the requirements `pip install -r requirements.txt`
run collect static `python manage.py collectstatic`

results: `django.core.exceptions.ImproperlyConfigured: You're using the staticfiles app without having set the STATIC_ROOT setting to a filesystem path.`

Current workaround fix django version to 5.0.0

**Comment by Artasov** [comment]
Fix it please!!! (5.1.4)

**Comment by theriverman** [comment]
@MaoMaoCake & @Artasov Sorry about the delay! Version 3.8.0 is now out and can be pulled from PyPI.

Please see the following block in the README:
https://github.com/theriverman/django-minio-backend/blob/9559249f380f895e325f89ad0eb7b84d8f5b51b5/README.md?plain=1#L113-L131

TL;DR: `STATICFILES_STORAGE` and `DEFAULT_FILE_STORAGE` were replaced by `STORAGES` in **Django 5.1**.
You must update your settings file accordingly by removing those entries and adding `STORAGES`.

**Comment by toabi** [comment]
It's weird… but I have no traces of `STATICFILES_STORAGE` in any of my settings… but when I configure STORAGES.staticfiles with Django 4.2.14 the STATICFILES_STORAGE get somehow set with the same value and django-minio-backend therefore fails with the implemented error.

**Comment by theriverman** [comment]
> It's weird… but I have no traces of `STATICFILES_STORAGE` in any of my settings… but when I configure STORAGES.staticfiles with Django 4.2.14 the STATICFILES_STORAGE get somehow set with the same value and django-minio-backend therefore fails with the implemented error.
> 
> Edit: Downgrade to 3.7.0 makes it work with django 4.2.14

I'm away from my computer but there's a WIP beta version of the package in GitHub (see tags). It's a big rework of the settings, so pay attention to the README. I think it's also available on PyPI if the beta version is requested explicitly.
